nf_core/components/nfcore_component.py

```diff
@@ -206,6 +206,7 @@ def get_inputs_from_main_nf(self) -> None:
             input_data = data.split("input:")[1].split("output:")[0]
             for line in input_data.split("\n"):
                 channel_elements: Any = []
+                line = line.split("//")[0]  # remove any trailing comments
```

Comment:
```suggestion
                line = line.split(" //")[0]  # remove any trailing comments
```
make it a bit more error proof

Reply:
Why do we care about leading spaces?  See:
https://github.com/nf-core/tools/blob/52e810986e382972ffad0aab28e94f828ffd509b/nf_core/modules/lint/main_nf.py#L577